tensorboard/plugins/text/summary_v2.py

```diff
@@ -26,10 +26,44 @@
 def text(name, data, step=None, description=None):
     """Write a text summary.
 
+    See also `tf.summary.scalar`, `tf.summary.SummaryWriter`, `tf.summary.image`.
+
+    Logs text Tensor values for later visualization and analysis in TensorBoard.  Like
+    `tf.summary.scalar` points, text points are each associated with a `step` and a
+    `name`.  All the points with the same `name` constitute a time series of text values.
+
+    For Example:
+    ```python
+    test_summary_writer = tf.summary.create_file_writer('test/logdir')
+    with test_summary_writer.as_default():
+        tf.summary.text('first_text', 'hello world!', step=0)
+        tf.summary.text('first_text', 'nice to meet you!', step=1)
+    ```
+
+    The text summary can also contain Markdown, and TensorBoard will render the text
+    as such.
+
+    ```python
+    with test_summary_writer.as_default():
+        table_as_markdown_text = '''
+              | *hello* | *there* |
+              |---------|---------|
+              | this    | is      |
+              | a       | table   |
+        '''
+        tf.summary.text('markdown_text', table_as_markdown_text, step=0)
+    ```
+
+    Since text is Tensor valued, each text point may be a Tensor of string values.
+    rank-1 and rank-2 Tensors are rendered as tables in TensorBoard.  For higher ranked
+    Tensors, you'll see just a 2D slice of the data.
```

Comment:
Maybe add "to avoid this, reshape the Tensor to at most rank-2 prior to passing it to this function."?